.github/workflows/sync-master-develop.yml

```
@@ -0,0 +1,16 @@
+name: Sync master/develop branches
+on:
+  pull_request:
+    types: [closed]
+    branches: master
```

@scurker suggested we use the workflow like this instead:

```suggestion
on:
  push:
    branches:
      - master
```

So rather than running when _any_ PR into `master` is closed, only run when a commit lands in `master`.